### ISSUE TITLE
can't locate node [ublox_gps] in package [ublox_gps]

### ISSUE THREAD

**Issue by WelsyCZ** [KumarRobotics/ublox#35]
EDIT: I wasn't running it under a catkin_workspace. My bad.

Hey, when I try to launch the package, I get this error:

`ERROR: cannot launch node of type [ublox_gps/ublox_gps]: can't locate node [ublox_gps] in package [ublox_gps]`

Here is my launch file:

```
<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <arg name="node_name" default="navilock_gpsnode"/>
  <arg name="param_file_name" doc="param file name" default="c94_m8p_base" />
  <arg name="output" default="screen" />
  <arg name="respawn" default="true" />
  <arg name="respawn_delay" default="30" />
  <arg name="clear_params" default="true" />

  <node pkg="ublox_gps" type="ublox_gps" name="$(arg node_name)"
        output="$(arg output)" 
        clear_params="$(arg clear_params)"
        respawn="$(arg respawn)" 
        respawn_delay="$(arg respawn_delay)">
    <rosparam command="load" 
              file="$(find ublox_gps)/config/$(arg param_file_name).yaml" />
  </node>
</launch>
```
And here is my config file (I know it's plain but Ive tried it with the premade ones, I dont think where the problem is, I havent even got to play with the config file as it has no effect atm)

```
# Navilock config

device: /dev/ttyACM0
frame_id: gps_base
dynamic_model: portable
nav_rate: 1
rate: 1

inf:
  all: true

publish:
  all: true

```

The device I have: https://www.navilock.de/produkte/G_61840/treiber.html
Basically rosmake goes well, compiles fine, but the launch won't ever work. And trying to use rosrun on the ublox_gps package won't even tab complete any node, like there weren't any.
